android/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why did you commit this file?

Reply:
As we've talked about it, this file is being generated all the time. Better to add it to our `.gitignore` project as I told you before